### ISSUE TITLE
About youtube API request restrictions

### ISSUE THREAD

**Issue by allenhaodfdfdf** [KaustubhPatange/YTPlayer#8]
I saw your project YTPlayer on github, and I was very surprised to have such a perfect project. I have a question below for consultation. Does this project use the official YouTube API? 

Is there a limit on the number of requests? I found another project on github to obtain data by parsing youtube website. Because there is no limit on the number of requests, this 

project is very popular on github. There are 7800 collections. I wonder if I can change this project to parse the website to obtain data. This is the link to that project: 

https://github.com/TeamNewPipe/NewPipe
If you can modify it I believe this will be a great project and look forward to your update

**Comment by KaustubhPatange** [comment]
Yes YTPlayer uses YouTube data API served from my php server (that data class is hidden though in this project due to security reasons).

There are limit to this requests right now it is overcome by supplying more api keys. 

Also I know about NewPipe project its library for parsing YouTube stream was perfect until recently it stopped working due to some reasons. 

If I got some time from my work, I will surely put a pull request to this project with an update ;)

**Comment by allenhaodfdfdf** [comment]
Looking forward to the update of TYPlayer.

**Comment by areyouoo** [comment]
Recently, I have been trying to transform him into a perfect project like NewPipe. But I have encountered a lot of technical problems. I can only expect you to complete this great project.

Please update as soon as possible! I believe he can become a Google-level project

**Comment by KaustubhPatange** [comment]
I can update YTPlayer whenever I encountered any issues so if there are issues in YTPlayer let me know.

About NewPipe project technically I am going through a lot these days (busy). I can only assure you but I can't guarantee a success!
